internal/repository/postgresdb/coreValues.go

```diff
@@ -20,113 +22,129 @@ func NewCoreValueRepo(db *sqlx.DB) repository.CoreValueStorer {
 	}
 }
 
-const (
-	listCoreValuesQuery  = `SELECT id, name, description, parent_core_value_id FROM core_values`
-	getCoreValueQuery    = `SELECT id, name, description, parent_core_value_id FROM core_values WHERE id = $1`
-	createCoreValueQuery = `INSERT INTO core_values (name,
-		description, parent_core_value_id) VALUES ($1, $2, $3) RETURNING id, name, description, parent_core_value_id`
-	//edit dele
-	deleteSubCoreValueQuery = `UPDATE core_values SET soft_delete = true, soft_delete_by = $1, updated_at = $2 WHERE org_id = $3 and parent_id = $4`
-	deleteCoreValueQuery    = `UPDATE core_values SET soft_delete = true, soft_delete_by = $1, updated_at = $2 WHERE org_id = $3 and id = $4`
-	//edit dele
-	updateCoreValueQuery = `UPDATE core_values SET (name, description) =
-		($1, $2) where id = $3 RETURNING id, name, description, parent_core_value_id`
-
-	// checkOrganisationQuery = `SELECT id from organizations WHERE id = $1`
-	checkUniqueCoreVal = `SELECT id from core_values WHERE name = $1`
-)
-
-func (cs *coreValueStore) ListCoreValues(ctx context.Context) (coreValues []dto.ListCoreValuesResp, err error) {
+func (cs *coreValueStore) ListCoreValues(ctx context.Context) (coreValues []repository.CoreValue, err error) {
+	queryBuilder := sq.Select("id", "name", "description", "parent_core_value_id").From("core_values")
+	listCoreValuesQuery, _, err := queryBuilder.ToSql()
+	if err != nil {
+		logger.Error(fmt.Sprintf("error in generating squirrel query, err: %s", err.Error()))
```

Comment:
return error without logging. you can wrap error here with `fmt.Errorf("failed to create query, err:%w", err)`